Doc/library/datetime.rst

```diff
@@ -317,6 +317,11 @@ Notes:
   >>> print(_)
   -1 day, 19:00:00
 
+(6)
+   The expression `t2 - t3` will always be equal to the expression `t2 + (-t3)` except
+   one condition when t3 is equal to `timedelta.max`, in that condition the former will produce result
```

Comment:
Are you sure you want to repeat the word "condition" here?  Why not simply "except when t3 is equal to `timedelta.max`, in that case ..."

Also, I feel you need an "a" before "result".

Reply:
point